tests/nnstreamer_sink/unittest_sink.cpp

```diff
@@ -1100,6 +1100,8 @@ TEST (tensor_sink_test, signals)
       (GCallback) _eos_cb, NULL);
   EXPECT_TRUE (handle_id > 0);
 
+  g_object_set (g_test_data.sink, "sync", FALSE, NULL);
```

Comment:
Or.. you can edit line 456:
```
-          "tensor_converter ! tensor_sink name=test_sink", option.num_buffers,
+          "tensor_converter ! tensor_sink sync=false name=test_sink", option.num_buffers,
```

Reply:
Plus there are so many similar cases if you look at line 456 and its neighbors.

Reply:
Yes, the meaning is the same. But, changing flags directly in `TEST_TYPE_VIDEO_RGB` might occur some problems because it's used in other unittests. Actually, other tests may not need this flag because they did not expect the exact number of buffer sinks.

Reply:
I added a common test option, `need_sync`.